tensorflow/core/kernels/concat_lib_gpu_impl.cu.cc

```diff
@@ -156,18 +161,19 @@ void ConcatGPUImpl(const Eigen::GpuDevice& gpu_device,
     // possibly due to decreasing occupancy
     // 4096 inputs is a lot, most code will take the smem path
     const int32 kMaxSmemBytesPerformance = 16384;
-    if (smem_usage < smem_max && smem_usage < kMaxSmemBytesPerformance)
-      TF_CHECK_OK(CudaLaunchKernel(concat_variable_kernel<T, IntType, true>,
-                                   config.block_count, config.thread_per_block,
-                                   smem_usage, gpu_device.stream(), input_ptrs,
-                                   output_scan, output->dimension(0),
-                                   output->dimension(1), output->data()));
-    else
-      TF_CHECK_OK(CudaLaunchKernel(concat_variable_kernel<T, IntType, false>,
-                                   config.block_count, config.thread_per_block,
-                                   0, gpu_device.stream(), input_ptrs,
-                                   output_scan, output->dimension(0),
-                                   output->dimension(1), output->data()));
+    if (smem_usage < smem_max && smem_usage < kMaxSmemBytesPerformance) {
+      TF_CHECK_OK(GpuLaunchKernel(
+          (concat_variable_kernel<T, IntType, true>), dim3(config.block_count),
```

Comment:
Remove extra parenthesis. Here and below.

Reply:
let me do that and push out a rebase